base/applications/control/control.c

```diff
@@ -7,6 +7,8 @@
  *              Copyright 2014 Herm�s B�lusca-Ma�to (hermes.belusca-maito@reactos.org)
  */
 
+#define COBJMACROS
+
 #include <stdio.h>
 
 #define WIN32_NO_STATUS
```

Comment:
```suggestion
#define WIN32_NO_STATUS
#define COBJMACROS

```

Reply:
No, I don't control <stdio.h>, what if it includes a Windows header?

I can move the WIN32_NO_STATUS to the top if that makes you feel better?

Reply:
If stdio or the other standard CRT headers were importing a windows header, they couldn't be used in places like the kernel or the bootloader. The fact that they can be used there shows that it's not the case.
(And you could just look at all the other examples in our codebase if you're not convinced.)